algorithms/CPlusPlus/Strings/remove-occurences.cpp

```diff
@@ -0,0 +1,48 @@
+/*
+Description: A C++ program to remove occurrence of characters of 2nd string from the 1st string
+
+*/
+
+#include <string>
+#include <iostream>
+using namespace std;
+
+string removeChars(string s1, string s2) {
+    
+    string out;
+    //checking if the character of 2nd string is present in the 1st string
```

Comment:
I believe you mean `checking the characters of the 1st string is present in the 2nd string`.
In line 15 you are looping through `s1` and in line 17 you are checking if the characters of 1st string is present
in `s2`, if you interchange those variables the end result will be different.
```suggestion
    //checking if the character of the 1st string is present in the 2nd string
```